### ISSUE TITLE
Make Travis automatically test point release branches

### ISSUE THREAD

**Issue by bmw** [certbot/certbot#4083]
Modifying .travis.yml every time is tedious.

**Comment by bmw** [comment]
To resolve this, `.travis.yml` should be modified so branches of the form `digit(s).digit(s).x` are tested in Travis. You can use [this page](https://docs.travis-ci.com/user/customizing-the-build#Building-Specific-Branches) as a reference.

**Comment by katebee** [comment]
Hello! 👋 I would like to volunteer for this

**Comment by bmw** [comment]
Great! Thanks for your interest in contributing.

The relevant file and lines are [here](https://github.com/certbot/certbot/blob/master/.travis.yml#L101). I expect we just need a simple regex matching what I wrote above, but I haven't looked closely at what is allowed by Travis.

**Comment by bmw** [comment]
@katebee, we did another release today and rather than [hard coding another branch](https://github.com/certbot/certbot/blob/0.10.x/.travis.yml#L99) I went ahead and made a PR for this issue. If you're still interested in contributing and want to work on test configuration, #4049 might be a good issue for you. If you'd rather work on a different issue and would like some guidance on what to pick, let me know what you're interested in and I can help you find something.
